### ISSUE TITLE
Why burp client is named "monitor"?

### ISSUE THREAD

**Issue by marcin-github** [CoffeeITWorks/ansible_burp2_server#72]
Hi!
I'd like to ask why client is named "monitor" instead hostname of server where burp server is installed? I can't modify settings of client `monitor`, I can't add includes etc. Maybe I have wrong assumption and server with burp server should be backuped in different way? But  it looks that ansible_burp2_client shouldn't be installed on the same host with ansible_burp2_server because it overwrite some files (like burp.conf) in `/etc/burp/`.

**Comment by pablodav** [comment]
Hello!

I didn't have time to reply properly but here is short answer :

Monitor is normally used as super client and in many cases by burp-ui agent
to be able to see and operate over every client (using burp server config
directives)

To backup your burp server makes sense yo use some external service or
another burp server and export data outside your burp server, could be to
external disk, tape remote storage, cloud, whatever.


More details will be added if needed.

**Comment by marcin-github** [comment]
Hi,
thank you for anwser.

> To backup your burp server makes sense yo use some external service or
another burp server and export data outside your burp server, could be to
external disk, tape remote storage, cloud, whatever.

Even if I want to do backup to external burp-server there is file collision with client configuration `/etc/burp/burp.conf` 
Also I think that backup on the same host has advantages, it's easy to go back and recover deleted/modified files on burp-server host.

**Comment by pablodav** [comment]
ok, you can manually setup another client or try to burp2_client role, first try in a test server as I have not tested it really but should work:

https://github.com/CoffeeITWorks/ansible_burp2_client/blob/master/defaults/main.yml#L5

Add these vars to your host_vars or group_vars or playbook:

´´´yaml
etc_dir: "/etc/burp-client"
# This with correct server name and IP, password, etc:
burp_client_server: "192.168.0.1"
burp_client_ssl_peer_cn: "burpserver"
burp_client_password: "password"
´´´

Also check others options you have in burp2_client setup.

So basically, you will setup a client in a different /etc directory than the burp server.

**Comment by pablodav** [comment]
In these steps I have separated the client and the monitor here: https://github.com/CoffeeITWorks/ansible_burp2_server/blob/master/tasks/10_local_client.yml#L21

So you will have to use burp -c /etc/burp/monitor.conf -a S
